db/migrations/1622554428_create_country_price_multipliers.up.sql

```diff
@@ -0,0 +1,261 @@
+CREATE TABLE IF NOT EXISTS country_price_multipliers(
+	id              SERIAL PRIMARY KEY,
+	country_code varchar(2) not null UNIQUE,
+	multiplier double precision NOT NULL,
+    updated_at timestamp NOT NULL default now()
+);
+
+CREATE INDEX IF NOT EXISTS country_price_multipliers_country_code_idx ON country_price_multipliers(country_code);
```

Comment:
This can be a unique index no? Or this column could just be PK

Reply:
While it could I'd rather it was not.